README.md

```diff
@@ -71,7 +71,7 @@ See `Ueberauth.Strategy` for more information on constructing the Ueberauth.Auth
 def application do
   # Add the application to your list of applications.
   # This will ensure that it will be included in a release.
-  [applications: [:logger, :ueberauth]]
+  [applications: [:logger, :oauth2, :ueberauth]]
```

Comment:
I'm in the process of updating all of the docs which should make this clearer.  Here's the updated README for the Facebook strategy: [uenberauth_facebook/README.md].(https://github.com/ueberauth/ueberauth_facebook/blob/doc-updates/README.md)